README.md

```diff
@@ -8,46 +8,69 @@ You can encrypt a file online at https://robinmoisson.github.io/staticrypt.
 
-The concept is simple but I am not a cryptographer, feel free to contribute or report any thought to the GitHub project! (Though be warned it might take me a long time to get to it - I apologize in advance)
+**Disclaimer:** The concept is simple and should work ok but I am not a cryptographer, if you have sensitive banking or crypto data you might want to use something else. :)
 
-Similar project: [MaxLaumeister/clientside-html-password](https://github.com/MaxLaumeister/clientside-html-password)
+You can report thoughts and issues to the [GitHub project](https://robinmoisson.github.io/staticrypt) but be warned I might be extremely slow to respond (though the community might help). If a serious security issue is reported I'll try to fix it quickly.
 
 ## CLI
 
-Staticrypt is available through npm as a CLI, install with `npm install -g staticrypt` and use as follow:
+Staticrypt is available through npm as a CLI, install with `npm install -g staticrypt` (with or without the `-g` flag) and use as follow:
 
     Usage: staticrypt <filename> <passphrase> [options]
 
     Options:
-      --help               Show help                                       [boolean]
-      --version            Show version number                             [boolean]
-      -e, --embed          Whether or not to embed crypto-js in the page (or use an
-                           external CDN)                   [boolean] [default: true]
-      -o, --output         File name / path for generated encrypted file
+      --help                    Show help                                   [boolean]
+      --version                 Show version number                         [boolean]
+      -e, --embed               Whether or not to embed crypto-js in the page (or use 
+                                an external CDN)
+                                                           [boolean] [default: true]
+      -o, --output              File name / path for generated encrypted file
                                                             [string] [default: null]
-      -t, --title          Title for output HTML page
+      -t, --title               Title for output HTML page
                                                 [string] [default: "Protected Page"]
-      -i, --instructions   Special instructions to display to the user.
+      -i, --instructions        Special instructions to display to the user.
                                                             [string] [default: null]
-      -f, --file-template  Path to custom HTML template with password prompt.
+      -f, --file-template       Path to custom HTML template with password prompt.
                               [string] [default: "[...]/cli/password_template.html"]
-
+      -r, --remember            Show a "Remember me" checkbox that will save the
+                                password in clear text in localStorage when
```

Comment:
I would add a very clear warning such as "WARNING: this will store your passwords in plain text on clients' web browsers!" so someone doesn't accidentally use this option and then get their password breached

Reply:
Specifically add info about the risk of a rogue extension or third-party JS reading your password that you mentioned in https://github.com/robinmoisson/staticrypt/issues/120#issuecomment-1033749495